### ISSUE TITLE
Support for Include

### ISSUE THREAD

**Issue by jygastaud** [quantumsheep/sshs#27]
Thanks for your tool.

It would be really good it can support `Include` (eg `Include conf.d/*`) pattern from `~/.ssh/config`.

**Comment by quantumsheep** [comment]
It's supposed to be already included, I'm using it locally. It doesn't work on your side?

**Comment by quantumsheep** [comment]
I didn't know `Include` directives could work inside `Host` directives. sshs is using [sshconfig](https://github.com/mikkeloscar/sshconfig), I implemented it limiting usage inside of `Host` directives (https://github.com/mikkeloscar/sshconfig/blob/9ddb34f89ae36bf3ab5662bcd715458a9c93a16e/parser.go#L149).

I'll try to fix this this week :)

**Comment by Gounick** [comment]
Hello,
I have the same usage and this error:

```
$ sshs
2022/06/15 09:08:14 include not allowed in Host block
```
```
$ cat ~/.ssh/config
Host *
  ForwardAgent yes

Include ~/.ssh/config.d/*.txt
```
```
$ sshs --version
sshs version 3.2.0
```
```
$ ssh -V
OpenSSH_8.2p1 Ubuntu-4ubuntu0.5, OpenSSL 1.1.1f  31 Mar 2020
```

**Comment by quantumsheep** [comment]
Hey! `Include` is now implemented in 4.0.0
